src/super_gradients/training/transforms/transforms.py

```diff
@@ -320,6 +320,41 @@ def check_valid_arguments(self):
         self.fill_mask, self.fill_image = _validate_fill_values_arguments(self.fill_mask, self.fill_image)
 
 
+class SegPadToDivisible(SegmentationTransform):
+    def __init__(self, divisible_value: int, fill_mask: int = 0, fill_image: Union[int, Tuple, List] = 0) -> None:
+        super().__init__()
+        self.divisible_value = divisible_value
+        self.fill_mask = fill_mask
+        self.fill_image = fill_image
+        self.fill_image = tuple(fill_image) if isinstance(fill_image, Sequence) else fill_image
+
+        self.check_valid_arguments()
+
+    def __call__(self, sample: dict):
+        image = sample["image"]
+        mask = sample["mask"]
+        w, h = image.size
+
+        padded_w = int(math.ceil(w / self.divisible_value) * self.divisible_value)
+        padded_h = int(math.ceil(h / self.divisible_value) * self.divisible_value)
+
+        # pad images from center symmetrically
+        if w != padded_w or h != padded_h:
+            padh = padded_h - h
+            padw = padded_w - w
+
+            image = ImageOps.expand(image, border=(0, 0, padw, padh), fill=self.fill_image)
```

Comment:
The comment says `pad images from center symmetrically`, while ImageOps.expand uses pad on right & bottom.

Reply:
Thanks, and sorry - [d3df33d](https://github.com/Deci-AI/super-gradients/pull/594/commits/d3df33d1126bd220fa6efa87b1fc3c0edc311f4c)